Docker/README.md

```diff
@@ -1,29 +1,64 @@
 # Docker Lab Contents
-This lab purpose is to install and use Docker to become familiar with Linux based containers and handle some of the common use cases around it.
 
-## Lab Writer and Trainer
-  - Bruno.Cornec@hpe.com
-  - Rene.Ribaud@hpe.com
-
-<!--- [comment]: # Table of Content to be added --->
+The goal of this lab is to install and use Docker to become familiar with Linux based containers and handle some of the common use cases around it. By the end of this lab you will have created a Web application comprised of a number of micro-services.
+
+## Lab Writers and Trainers
+  - bruno.cornec@hpe.com
+  - rene.ribaud@hpe.com
+
+<!-- TOC -->
+
+- [Docker Lab Contents](#docker-lab-contents)
+    - [Lab Writers and Trainers](#lab-writers-and-trainers)
+    - [Objectives of the Docker Lab](#objectives-of-the-docker-lab)
+    - [Reference documents](#reference-documents)
+    - [Note on Linux commands](#note-on-linux-commands)
+- [Environment setup](#environment-setup)
+    - [Proxy consideration](#proxy-consideration)
+    - [Docker installation](#docker-installation)
+        - [Ubuntu installation](#ubuntu-installation)
+        - [RHEL installation](#rhel-installation)
+        - [Check installation](#check-installation)
+- [Using Docker](#using-docker)
+    - [The first container](#the-first-container)
+    - [The second container](#the-second-container)
+- [Configuring owncloud in a container](#configuring-owncloud-in-a-container)
+- [Using Docker compose](#using-docker-compose)
+    - [Installing Docker compose](#installing-docker-compose)
+    - [Our first Docker-compose.yml file](#our-first-docker-composeyml-file)
+    - [Going further with Docker-compose.yml](#going-further-with-docker-composeyml)
+- [Using Docker-machine to create Docker hosts](#using-docker-machine-to-create-docker-hosts)
+- [Using Docker Swarm](#using-docker-swarm)
+    - [Installing Docker Swarm](#installing-docker-swarm)
+    - [Installing on RHEL 7](#installing-on-rhel-7)
+    - [Installing on Ubuntu](#installing-on-ubuntu)
+    - [Installing the engine manually](#installing-the-engine-manually)
+    - [Installing the engine in the Cloud](#installing-the-engine-in-the-cloud)
+    - [Using Docker Swarm to make our configuration available and scalable](#using-docker-swarm-to-make-our-configuration-available-and-scalable)
+        - [CentOS/RHEL](#centosrhel)
+        - [Ubuntu](#ubuntu)
+- [Deploy a cloud native application.](#deploy-a-cloud-native-application)
+    - [Objectives](#objectives)
+
+<!-- /TOC -->
 
```

Comment:
Switched ToC creation to [gh-md-toc](https://github.com/ekalinin/github-markdown-toc) rather than editor specific editor.
Does this help?